Game/Levels/AdvMultiplication/L06mul_right_eq_one.lean

```diff
@@ -74,7 +74,7 @@ Statement mul_right_eq_one (x y : ℕ) (h : x * y = 1) : x = 1 := by
   exact one_ne_zero
   Hint (hidden := true) "Now you can `apply le_mul_right at h2`."
   apply le_mul_right at h2
-  Hint (hidden := true) "Now `rw [h] at h2` so you can `apply le_one at hx`."
+  Hint (hidden := true) "Now `rw [h] at h2` so you can `apply le_one at h2`."
```

Comment:
```suggestion
  Hint (hidden := true) "Now `rw [{h}] at {h2}` so you can `apply le_one at {h2}`."
```

Reply:
`{ }` is used for variable names in hints so that the hint shows the actual name the player chose. This should be done in all hints but is still very inconsistent throughout this game